src/node-lib/curriculum-api-2023/queries/specialistLessonOverview/specialistLessonOverview.schema.ts

```diff
@@ -0,0 +1,123 @@
+import { z } from "zod";
+
+import { LessonOverviewPageData } from "../lessonOverview/lessonOverview.schema";
+import {
+  baseLessonOverviewSchema,
+  lessonOverviewQuizData,
+} from "../../shared.schema";
+
+export type SpecialistLessonOverviewData = Omit<
+  LessonOverviewPageData,
+  | "keyStageSlug"
+  | "keyStageTitle"
+  | "tierTitle"
+  | "tierSlug"
+  | "yearTitle"
+  | "examBoardTitle"
+> & {
+  developmentStageTitle: string;
+  developmentStageSlug: string;
+  phaseTitle: string;
+  phaseSlug: string;
+  isCanonical?: boolean;
+  threads?: Threads[] | null;
+};
+
+const content_guidance_schema = z.object({
+  content_guidance_label: z.string(),
+  content_guidance_description: z.string(),
+  content_guidance_area: z.string(),
+});
+
+const misconceptions_and_commom_mistakes_schema = z.object({
+  misconception: z.string(),
+  response: z.string(),
+});
+
+const teacher_tips_schema = z.object({
+  teacher_tip: z.string(),
+});
+
+const lesson_equipment_and_resources_schema = z.object({
+  equipment: z.string(),
+});
+
+const key_learning_points_schema = z.object({
+  key_learning_point: z.string().nullable(),
+});
+
+const threads = z.object({
+  threadSlug: z.string(),
+  threadTitle: z.string(),
+});
+
+type Threads = z.infer<typeof threads>;
+
+const combined_programme_fields = z.object({
+  subject: z.string(),
+  subject_slug: z.string(),
+  developmentstage: z.string(),
```

Comment:
developmentstage and phase need to be nullable, see `specialistLessonDownloadRawSchema`

Reply:
Fixed ✅